Razan_Almusa/week_09/day_01/ReactWeather/app/javascript/components/App.js

```diff
@@ -0,0 +1,55 @@
+import React from "react"
+import PropTypes from "prop-types"
+import Search from "./Search";
+import WeatherResult from "./WeatherResult";
+
+
+
+class App extends React.Component {
+
+  constructor() {
+    super();
+    this.state = { data: null };
+    this.search = this.search.bind(this);
+  }
+
+  search(term) {
+
+    fetch(`http://api.openweathermap.org/data/2.5/weather?q=${term}&APPID=401e0cd27bd006446f78b068ab41d26a`)
+      //https://openweathermap.org/?apikey=401e0cd27bd006446f78b068ab41d26a=${term}`,{mode:'no-cors'})
+      .then(r => r.json())
+      .then(data => {
+        this.setState({ data: data });
+        console.log(this.state.data.name);
+        console.log(this.state.data.main.temp);
+        console.log(this.state.data.weather[0].description);
```

Comment:
You don't need to log data here after you test your request, unless you really want to of course.